CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+* Breaking Changes
+  * DropInRequest
+    * Remove `collectDeviceData` from `DropInRequest` in favor of always returning device data
```

Comment:
Maybe we update this to mention where the device data gets returned (BTDropInResult)?

Reply:
Yeah that sounds more clear - updated.